business_rules/operators.py

```diff
@@ -938,6 +939,67 @@ def is_ordered_by(self, other_value: dict) -> pd.Series:
     def is_not_ordered_by(self, other_value: dict) -> pd.Series:
         return ~self.is_ordered_by(other_value)
 
+    @type_operator(FIELD_DATAFRAME)
+    def target_is_sorted_by(self, other_value: dict) -> pd.Series:
+        """
+         Checking if  SESEQ is  sorted based on SESTDTC within USUBJID
+        """
+
+        target: str = self.replace_prefix(other_value.get("target"))
+        within: str = self.replace_prefix(other_value.get("within"))
+        columns = other_value["comparator"]
+
+        for col in columns:
+            comparator: str = self.replace_prefix(list(col.items())[0][0])
+            params = list(col.items())[0][1]
```

Comment:
Shouldn't each col in columns be a dict of the form:
{"name": <Var name>, "sort_order": "ASC" or  "DESC", "null_position": "first" or "last"}

If so you can get the values by  their keys instead of using indexing